stable/aws-pod-identity-webhook/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "aws-pod-identity-webhook.fullname" . }}
+  labels:
+{{ include "aws-pod-identity-webhook.labels" . | indent 4 }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    prometheus.io/port: "443"
+    prometheus.io/scheme: "https"
```

Comment:
Thanks so much for your work on this chart.
For Prometheus scraping to work in my environment I had the change these lines to the following:
```
    prometheus.io/port: "9999"
    prometheus.io/scheme: "http"
```
If they are different depending on how Prometheus has been deployed into the cluster, perhaps these need to be defined in the `stable/aws-pod-identity-webhook/values.yaml` file and references here so that they can be overridden locally?

Reply:
Cool. Feel free to make a PR after it's merged.